lib/redirect.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>ArgumentParser Documentation</title>
```

Comment:
Shouldn't this be AsyncAlgorithms Documentation?

Reply:
oof oops, teaches me to copy paste :P